docs/notebooks/using_mclip.md

```diff
@@ -0,0 +1,229 @@
+
+![WandB-mclip](images/WandB-mclip.png)
+
+<!-- #endregion -->
+
+## Saving
+
+After the run has finished successfully, you can download the tuned model on your local machine:
+
+```python
+artifact = run.save_artifact('mclip-model')
+```
+
+## Inference
+
+Now you saved the `artifact` into your host machine,
+let's use the fine-tuned model to encode a new `Document`:
+
+```python
+text_da = DocumentArray([Document(text='setwas Text zum Codieren')])
+image_da = DocumentArray([Document(uri='https://upload.wikimedia.org/wikipedia/commons/4/4e/Single_apple.png')])
+
+mclip_text_encoder = finetuner.get_model(artifact=artifact, select_model='clip-text')
+mclip_image_encoder = finetuner.get_model(artifact=artifact, select_model='clip-vision')
+
+finetuner.encode(model=mclip_text_encoder, data=text_da)
+finetuner.encode(model=mclip_image_encoder, data=image_da)
+
+print(text_da.embeddings.shape)
+print(image_da.embeddings.shape)
+```
+
+<!-- #region -->
+```bash
+(1, 512)
+(1, 512)
+```
+
+```{admonition} what is select_model?
+When fine-tuning CLIP, we are fine-tuning the CLIPVisionEncoder and CLIPTextEncoder in parallel.
+The artifact contains two models: `clip-vision` and `clip-text`.
+The parameter `select_model` tells finetuner which model to use for inference, in the above example,
+we use `clip-text` to encode a Document with text content.
+```
+
+```{admonition} Inference with ONNX
+In case you set `to_onnx=True` when calling `finetuner.fit` function,
+please use `model = finetuner.get_model(artifact, is_onnx=True)`
+```
+<!-- #endregion -->
+
+## Before and After
+We can directly compare the results of our fine-tuned model with an untained multilingual clip model by displaying the matches each model has for the same query:
+
+```python
+from finetuner import build_model
+
+pt_query = copy.deepcopy(query_data)
+pt_index = copy.deepcopy(index_data)
+
+ft_query = copy.deepcopy(query_data)
+ft_index = copy.deepcopy(index_data)
+
+zero_shot_text_encoder = build_model(
+    name = 'xlm-roberta-base-ViT-B-32::laion5b_s13b_b90k',
+    select_model = 'clip_text',
+)
+zero_shot_image_encoder = build_model(
+    name = 'xlm-roberta-base-ViT-B-32::laion5b_s13b_b90k',
+    select_model = 'clip_vision',
+)
+
+finetuner.encode(model=zero_shot_text_encoder, data=pt_query)
+finetuner.encode(model=zero_shot_image_encoder, data=pt_index)
+
+finetuner.encode(model=mclip_text_encoder, data=ft_query)
+finetuner.encode(model=mclip_image_encoder, data=ft_index)
+
+pt_query.match(pt_index)
+ft_query.match(ft_index)
+
+print('results for query: "externe mikrofone (external microphone)" using a zero-shot model (top) and the fine-tuned model (bottom)')
+query_pt[72].matches[0].plot_image_sprites()
+query_ft[72].matches[0].plot_image_sprites()
+
+print('results for query: "prozessorlüfter (processor fan)" using a zero-shot model (top) and the fine-tuned model (bottom)')
+query_pt[189].matches[0].plot_image_sprites()
+query_ft[189].matches[0].plot_image_sprites()
+
```

Comment:
can you add the images here as markdown cells otherwise they are not visible in the documentation